.github/workflows/python-app.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest psycopg2
```

Comment:
instead of install by pip, remove that line and create a requirements.txt with dependencies